modules/05_add_blob_storage/src/blob_storage.py

```diff
@@ -0,0 +1,41 @@
+import os
+import uuid
+from azure.storage.blob import BlobServiceClient
+import logging
+
+logging.getLogger('azure').setLevel(logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+class BlobStorage:
+    """
+        Blob store to store secret words.
+    """
+
+    def __init__(self):
+        connection_string = os.getenv('AZURE_STORAGE_CONNECTION_STRING', '')
+        self.client = BlobServiceClient.from_connection_string(connection_string)
+        container_name = 'images-to-guess'
+        self.container_client = self.client.get_container_client(container_name)
+        try:
+            self.container_client.create_container(timeout=1000)
+        except Exception as exp: # pylint: disable=broad-except
+            print("Exception trying to create container {}".format(exp))
```

Comment:
```suggestion
    def __init__(self, container_client):
        self.container_client = container_client
            
    def from_env(cls):
        connection_string = os.getenv('AZURE_STORAGE_CONNECTION_STRING', '')
        client = BlobServiceClient.from_connection_string(connection_string)
        container_name = 'images-to-guess'
        self.container_client = client.get_container_client(container_name)
        try:
            container_client = container_client.create_container(timeout=1000)
        except Exception as exp: # pylint: disable=broad-except
            print("Exception trying to create container {}".format(exp))
      return cls(container_client)
```

Reply:
I managed to test without classmethod, which requires advanced knowledge. What do you think?

Reply:
Imho doing monkeypatching properly is far more advanced than a classmethod :)